scripts/push_coverity_scan.py

```diff
@@ -0,0 +1,454 @@
+#!/usr/bin/env python3
```

Comment:
We're going to run this script at least on both tf-psa-crypto and mbedtls, and possibly on LTS branches. So it belongs in `mbedtls-test` or `mbedtls-framework`. Probably `mbedtls-test` since this isn't something we normally run manually?

Reply:
I was in discussion with Bence about whether or not this belonged in mbedtls-test anyway. Given it won't pass on Ubuntu16.04, I think that its probably a good idea to move it there, but need to have a quick word with Bence first (he's not in today)